src/fides/api/alembic/migrations/versions/32497f08e227_translation_cleanup.py

```diff
@@ -0,0 +1,380 @@
+"""translation_cleanup Cleans up deprecated tables and fields after multitranslation migration
+
+Revision ID: 32497f08e227
+Revises: 0c65325843bd
```

Comment:
Downrev will need to be repointed to Adam's  data migration https://github.com/ethyca/fides/pull/4636/files

Reply:
When Adam's data migration and my followup schema migration changes are in, we should re-deploy this on a test environment with a fresh db I'd think.